frontend/packages/dev-console/src/components/import/pipeline/PipelineSection.tsx

```diff
@@ -2,20 +2,49 @@ import * as React from 'react';
 import { connectToFlags, FlagsObject } from '@console/internal/reducers/features';
 import { getBadgeFromType } from '@console/shared';
 import { Split, SplitItem, Alert } from '@patternfly/react-core';
+import { getActiveNamespace } from '@console/internal/actions/ui';
+import { useAccessReview } from '@console/internal/components/utils';
 import { useFormikContext, FormikValues } from 'formik';
-import { PipelineModel } from '../../../models';
-import { FLAG_OPENSHIFT_PIPELINE } from '../../../const';
+import { PipelineModel, PipelineResourceModel } from '../../../models';
+import { FLAG_OPENSHIFT_PIPELINE, CLUSTER_PIPELINE_NS } from '../../../const';
 import FormSection from '../section/FormSection';
 import PipelineTemplate from './PipelineTemplate';
 
 type PipelineSectionProps = {
   flags: FlagsObject;
 };
 
+const useHasCreatePipelineAccess = (): boolean => {
+  const canListPipelines = useAccessReview({
+    group: PipelineModel.apiGroup,
+    resource: PipelineModel.plural,
+    namespace: CLUSTER_PIPELINE_NS,
+    verb: 'list',
+  });
+
+  const canCreatePipelines = useAccessReview({
+    group: PipelineModel.apiGroup,
+    resource: PipelineModel.plural,
+    namespace: getActiveNamespace(),
+    verb: 'create',
+  });
+
+  const canCreatePipelineResource = useAccessReview({
+    group: PipelineResourceModel.apiGroup,
+    resource: PipelineResourceModel.plural,
+    namespace: getActiveNamespace(),
+    verb: 'create',
+  });
+
+  return canListPipelines && canCreatePipelines && canCreatePipelineResource;
+};
+
 const PipelineSection: React.FC<PipelineSectionProps> = ({ flags }) => {
   const { values } = useFormikContext<FormikValues>();
 
-  if (flags[FLAG_OPENSHIFT_PIPELINE]) {
+  const canCreate = useHasCreatePipelineAccess();
```

Comment:
or may be: 
```suggestion
  const hasCreatePipelineAccess = usePipelineAccessReview();
```

Reply:
Agreed, this is cleaner. Everything else looks good to me, just would like this change as well before providing approval.

Reply:
@debsmita1 @andrewballantyne Done.